README.md

```diff
@@ -1,2 +1,64 @@
 # benchcheck
-Benchcheck runs benchmarks on different versions of a Go module  and compares statistics accross them, failing on performance regressions.
+
+**benchcheck** runs benchmarks on two different versions of a Go module and
+compares statistics across them, failing on performance regressions.
+
+This tool aims at being Go specific and wildly simple, leveraging standard Go
+benchmarks + [benchstat](https://pkg.go.dev/golang.org/x/perf/cmd/benchstat),
+composing them to provide just a few extra features:
+
+* Given two module versions of a package, it automatically benchmarks and compares them
+* Given a threshold on the delta of performance loss, the check fails (useful for CI's)
+
+And that is it, no more, no less.
+
+# Dependencies
+
+* git
+* go
+
+# Install
+
+Just run:
+
+```sh
+go install github.com/madlambda/benchcheck/cmd/benchcheck@latest
+```
+
+# Usage
+
+For details just run:
+
+```sh
+benchcheck -help
+```
+
+By default all benchmarks are run with -benchmem, so you also
+have information on memory allocations.
+
+Comparing performance between two versions of a Go module
+and just showing results on output (no check performed):
+
+```
+benchcheck cool.go.module v0.0.1 v0.0.2
```

Comment:
what we're going to do if the two versions have a different set of tests? show comparison for the ones that match and ignore the others?

Reply:
I think so, it is how benchstat behaves (AFAIK)